example/swagger-files/anyOf-example.json

```
@@ -21,13 +21,20 @@
                     "title": "Pet by Type",
                     "type": "object",
                     "properties": {
-                      "pet_type": {
+                      "_id": {
+                        "pattern": "^[a-fA-F\\d]{24}$",
+                        "example": "000000000000000000000000",
                         "type": "string"
                       },
+                      "pet_type": {
+                        "type": "string",
+                        "example": "bassotto"
```

```suggestion
                        "example": "dachshund"
```

maybe in english could be better :)